catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/endpoint/query/OpenSearchQuery.java

```diff
@@ -255,11 +273,10 @@ public void addTemporalFilter(String dateStart, String dateEnd, String dateOffse
 
     // If either start date OR end date is specified and non-empty, then
     // a temporal filter can be created
-    if ((dateStart != null && !dateStart.trim().isEmpty())
-        || (dateEnd != null && !dateEnd.trim().isEmpty())) {
-      temporalFilter = new TemporalFilter(dateStart, dateEnd);
-    } else if (dateOffset != null && !dateOffset.trim().isEmpty()) {
-      temporalFilter = new TemporalFilter(Long.parseLong(dateOffset));
+    if (StringUtils.isNotBlank(dateStart) || StringUtils.isNotBlank(dateEnd)) {
+      temporalFilter = new TemporalFilter(dateStart.trim(), dateEnd.trim());
```

Comment:
can dateStart or dateEnd be null here? Just thinking that previously we were checking for null and if one of them isn't null and one is then we would get a NPE on one of the trim calls.